GitUI/Editor/FileViewer.cs

```diff
@@ -20,6 +20,7 @@
 using GitUIPluginInterfaces;
 using JetBrains.Annotations;
 using ResourceManager;
+using SmartFormat;
```

Comment:
What's this used for?

Reply:
```
private readonly TranslationString _cannotViewImage = new TranslationString("Cannot view image {0}");
<snip>
internalFileViewer.SetText(_cannotViewImage.Text.FormatSmart(fileName), openWithDifftool);
```

Reply:
Thank you

> _cannotViewImage.Text.FormatSmart(fileName)

What does this do?
Why can't we use the standard BCL function `string.Format`?

Reply:
> Why can't we use the standard BCL function string.Format

Because I preferred the compact format. I could change to:
```
internalFileViewer.SetText(string.Format(_cannotViewImage.Text, fileName), openWithDifftool);
                                    
```